src/elements/select/select.ts

```diff
@@ -141,6 +146,14 @@ export class SbbSelectElement extends SbbUpdateSchedulerMixin(
     );
   }
 
+  private _onSelectAttributesChange(mutationsList: MutationRecord[]): void {
```

Comment:
@kyubisation Should we implement this helper logic or wait for native form support (solves the accessibility issue of linking the <label> to the select)

Reply:
Refinement 09/11:
the solution is temporary acceptable, because it fixes a wrong behavior. 
A couple of `TODO`s have been added and the whole logic will be possibly removed with the merge of story #3062